src/backend/metal/src/internal.rs

```diff
@@ -451,7 +451,10 @@ pub struct ServicePipes {
 
 impl ServicePipes {
     pub fn new(device: &metal::DeviceRef) -> Self {
+        #[cfg(target_os = "macos")]
         let data = include_bytes!("./../shaders/gfx_shaders.metallib");
```

Comment:
nit: we could also do `if cfg!(target_os = "macos") { "../shaders/gfx_shaders.metallib" } else { "../shaders/gfx_shaders_ios.metallib" }` as an expression

Reply:
@kvark how i can implement this?

If i do this:
```rust
let data = if cfg!(target_os = "macos") {
            include_bytes!("./../shaders/gfx-shaders-macos.metallib")
        } else {
            include_bytes!("./../shaders/gfx-shaders-ios.metallib")
        };
```
I get compiler error - `if and else have incompatible types`

Reply:
I guess it would have to be `&include_bytes!(...)[..]`